public/index.html

```diff
@@ -107,37 +110,37 @@ <h1 id="tagline">We study ecosystem change where land meets sea at the top of th
             <button class="btn btn-light btn-lg m-1"><a href="#news">Project news</a></button>
             <br>
             <br>
-            <p>* We joined the <a class="text-white" style="text-decoration: underline;" rel="external noopener" href="https://lternet.edu/" target="_blank">U.S. LTER network</a> in 2017. The northernmost in the 28-site network, BLE
+            <p>* We joined the <a class="text-white" style="text-decoration: underline;" rel="external noopener" href="https://lternet.edu/" target="_blank" aria-label="LTER network">U.S. LTER network</a> in 2017. The northernmost in the 28-site network, BLE
             LTER will address gaps in our understanding of the rapidly changing Arctic. We collaborate
          closely with other sites to do the best science and to promote open research.</p>
       </div>
    </div>
    <div class="container container-small my-5 py-5">
-      <h1 class="text-center" id="numbers">By the numbers</h1>
+      <h2 class="h1 text-center" id="numbers">By the numbers</h2>
       <br>
       <div id="glance">
          <div class="row lead text-white justify-content-center">
             <div class="col-md-5 mb-3 mr-md-3 p-5 text-right" style="background: linear-gradient(rgba(0,0,0,0.4), rgba(0,0,0,0.4)), url(img/12pi.jpg); background-size: cover; background-position: center;">
-               <h1>12</h1>
-               <p class="h3">investigators</p>
-               <p class="h3">from six universities</p>
+               <h3 class="h2">12</h3>
+               <h4 class="h4">investigators</h4>
+               <h4 class="h4">from six universities</h4>
```

Comment:
Did you mean to make the text smaller?

Reply:
Also, these aren't really headings. I think they should all be <p>s with the appropriate heading class.

Reply:
what's "s with the appropriate heading class"? do you mean <p>?

Reply:
yea, removing the bootstrap class and adding our own (but with essentially the bootstrap class style) gets dinged too. 

my idea: I found out that font-weight seems to be the turnkey, removing it means we're clear. Also, we can get around the multiple `<p>` elements that by themselves don't make sense by adding line breaks, like this: 

```
<p style="font-size: 2rem; line-height: 1.5; margin-bottom: .5rem;">
               12 <br>
               investigators <br>
               from six universities</p>
```
We don't get differentiation between the lines but that's ok with me.